pkg/apis/core/types.go

```diff
@@ -2159,6 +2161,10 @@ type Nic struct {
 	// default to false
 	// +optional
 	Ipv6Enabled bool
+	// To provision the server instance with a NIC for a network, specify the UUID of the network in the uuid attribute in a networks object
+	// Required if omit the port attribute
+	// +optional
+	Uuid string
```

Comment:
which component to assign the value for the UUID field ?

Reply:
In OpenStack, network id is required if omit the port field to create an instance.
[OpenStack Link](https://docs.openstack.org/api-ref/compute/?expanded=create-server-detail#create-server)

![image](https://user-images.githubusercontent.com/19650434/88747551-d365e980-d103-11ea-94c9-88904f621b0c.png)
In the image above, it is a yaml file I use to specify nic uuid in order to create an instance in OpenStack